app/views/layouts/application.html.erb

```diff
@@ -12,6 +12,25 @@
   </head>
 
   <body>
+    <% if %w[faculties students].any? { |i| i == controller.controller_name } %>
+      <br>
+      <div class="row">
+        <div class="col-3"></div>
+        <div class="col-6 rounded" style="border: 3px solid #007fff!important">
+            <h3 class="text-primary text-center">-: Callbacks :-</h3>
+            <% $callback.each do |k, v| %>
+              <hr><h5><strong><%= k %> method:</strong></h5>
+              <ul>
+                <% $callback[k].each do |value| %>
+                  <li><%= value.keys[0] %> : <%= value.values[0] %></li>
+                <% end %>
+              </ul>
+            <% end %>
+        </div>
+        <div class="col-3"></div>
+      </div>
```

Comment:
Create a folder under view name shared, create a partial file inside a folder, and move this common code into that partial.
Call that partial in application.html.erb